kernel/locking/lockdep.c

```diff
@@ -755,6 +756,9 @@ static int static_obj(const void *obj)
 	if (arch_is_kernel_initmem_freed(addr))
 		return 0;
 
+	if (is_kfence_address(obj))
```

Comment:
Can you please add a comment here?
Something along the lines of "Lockdep complains if certain structures are in static data. Exclude kfence memory." should go.

Reply:
The function 'static_obj' only checks if something is a static object. I added the appropriate comment.